src/dz08/FileLoggerConfigurationLoader.java

```diff
@@ -0,0 +1,28 @@
+package dz08;
+import org.json.simple.JSONObject;
+import  org.json.simple.parser.*;
+
+import java.io.FileReader;
+import java.io.IOException;
+
+public class FileLoggerConfigurationLoader {
+    public void FileConfiguration() throws IOException, ParseException {
```

Comment:
It is a very bad practice to make a constructor for one class in another. Remove it. The method to load the config should name "load"